### ISSUE TITLE
Focus indicator on TooltipIcon missing when TABing to it on Safari

### ISSUE THREAD

**Issue by daniel-schulz-privat** [carbon-design-system/carbon#7257]
## Focus indicator missing
When navigating to a tooltip icon using the keyboard, there is no focus indicator shown on the icon.

## Environment
**OS:** MacOS Catalina 10.15.7
**Browser:** Safari Version 14.0 (15610.1.28.1.9, 15610)
**AT:** Keyboard navigation
**Carbon version:** The version that is currently on display with https://www.carbondesignsystem.com

## Steps to reproduce the issue
1. open Safari on MacOS
2. Navigate to https://www.carbondesignsystem.com/components/tooltip/usage
3. In the live-demo section, select 'Icon Tooltip' variant
4. Navigate to the icon in the live-demo section using keybord only (TAB key)

**Expected:** The focus indicator is shown to indicate that the icon has focus.
**Actual:** No focus indicator is shown.

Note that both Chrome and Firefox do show the focus border as expected - at least for me on MacOS.

## Additional information
1. Firefox does show a focus indicator, i.e. the blue border (see green box in below)
<img width="1080" alt="Screenshot 2020-11-10 at 13 48 30" src="https://user-images.githubusercontent.com/20154051/98677690-a5787e00-235d-11eb-905e-5ac4fc0411b0.png">

2. Safari does not show a focus indicator (see red box below)
<img width="1080" alt="Screenshot 2020-11-10 at 13 49 24" src="https://user-images.githubusercontent.com/20154051/98677786-d22c9580-235d-11eb-8075-cb429b47fced.png">

**Comment by emyarod** [comment]
I believe this is an issue with Safari ignoring tabindex

related https://github.com/carbon-design-system/carbon-website/issues/1944 https://github.com/carbon-design-system/carbon/issues/6444

**Comment by tw15egan** [comment]
Confirmed focus styles appear if the new safari setting is enabled:

<img width="406" alt="Screen Shot 2020-11-10 at 12 00 09 PM" src="https://user-images.githubusercontent.com/11928039/98727130-50803c00-234c-11eb-8ceb-71662bedc0ba.png">


<img width="682" alt="Screen Shot 2020-11-10 at 11 59 29 AM" src="https://user-images.githubusercontent.com/11928039/98727060-3181aa00-234c-11eb-836c-a293354fc471.png">

**Comment by daniel-schulz-privat** [comment]
Thank you guys. Somehow I totally missed that option.

**Comment by tw15egan** [comment]
@daniel-schulz-privat no worries, it seems like its a common issue and was somewhat recently introduced 👍
